<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastStart</class>
 <widget class="QDialog" name="FastStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>571</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutIcon">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../merit.qrc">:/icons/merit</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>15</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stack">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>119</y>
     <width>601</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="choicePage">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>581</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#384c62;&quot;&gt;How you want to download&lt;br/&gt;the Merit Blockchain?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="snapshotButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>221</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Sans Serif&quot;; color: rgb(32, 103, 209);</string>
     </property>
     <property name="text">
      <string>QuickStart</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="peersButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>221</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Sans Serif&quot;; color: rgb(32, 103, 209);</string>
     </property>
     <property name="text">
      <string>Standard</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#384c62;&quot;&gt;Download a snapshot of the blockchain, and then get the most recent transactions from Community Peers.&lt;br/&gt;&lt;br/&gt;This is the fastest option.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>290</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#384c62;&quot;&gt;Download only from peers in the Merit Community.&lt;br/&gt;&lt;br/&gt;This is slower, but more decentralized.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="downloadPage">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>541</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#384c62;&quot;&gt;A Snapshot of the blockchain is being downloaded&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#384c62;&quot;&gt;from the Merit Foundation!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="margin">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#7a90a7;&quot;&gt;Progress&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QStackedWidget" name="overviewSlides">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>179</y>
       <width>460</width>
       <height>224</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>460</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>460</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="communityPage">
      <widget class="QLabel" name="communityIcon">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>101</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../merit.qrc">:/icons/community</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="communityLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>461</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#2067d1;&quot;&gt;COMMUNITY&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;The world's first invite-only digital currency;&lt;br/&gt; with better incentives for everyone.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="safetyPage">
      <widget class="QLabel" name="safetyIcon">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>101</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../merit.qrc">:/icons/safety</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="safetyLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>461</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#2067d1;&quot;&gt;SAFETY&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#292929;&quot;&gt;Decentralized vaults, password-protected&lt;br/&gt; transactions, and cancellable transactions.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="simplicityPage">
      <widget class="QLabel" name="safetyIcon_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>101</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../merit.qrc">:/icons/simplicity</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="safetyLabel_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>461</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#2067d1;&quot;&gt;SIMPLICITY&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Send Merit as easy as a text message or email,&lt;br/&gt;whether recipients are on the network or not.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="waitPage">
      <widget class="QPushButton" name="hourglassIcon">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../merit.qrc">
         <normaloff>:/icons/hourglass</normaloff>
         <disabledoff>:/icons/hourglass</disabledoff>:/icons/hourglass</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="infoText_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>461</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#2067d1;&quot;&gt;HANG TIGHT&lt;br/&gt;&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;Before you can unlock your wallet, &lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;this application&lt;br/&gt;has to finish syncing with other peers on the network.&lt;br/&gt;Merit is a decentralized network that is powered&lt;br/&gt;by people like you.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <zorder>stack</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources>
  <include location="../merit.qrc"/>
 </resources>
 <connections/>
</ui>
